docs/condition.md

```diff
@@ -54,6 +59,10 @@ rules:
             - "integration_v[0-9]{2}$"
         comments:
           count: ">10"
+        patch:
```

Comment:
Probably need to check only for new files.

Reply:
Was my first assumption as well, but what if the PR contains a _removal_ of important code?